dependencies.yaml

```
@@ -836,12 +837,17 @@ dependencies:
           - msgpack-python
           - moto>=4.0.8
           - s3fs>=2022.3.0
+          - python-xxhash
       - output_types: pyproject
         packages:
           - msgpack
           - &tokenizers tokenizers==0.15.2
           - &transformers transformers==4.39.3
           - tzdata
+          - xxhash
+      - output_types: requirements
+        packages:
+          - xxhash
```

Usually we combine `pyproject` and `requirements`. I'm not sure why this is not doing that already...
```suggestion
      - output_types: [pyproject, requirements]
        packages:
          - msgpack
          - &tokenizers tokenizers==0.15.2
          - &transformers transformers==4.39.3
          - tzdata
          - xxhash
```